charts/k8s-infra/templates/_config.tpl

```diff
@@ -129,8 +131,8 @@ Build config file for deployment OpenTelemetry Collector: OtelDeployment
 
 {{- define "opentelemetry-collector.applyClusterMetricsConfig" -}}
 {{- $config := mustMergeOverwrite (include "opentelemetry-collector.clusterMetricsConfig" .Values | fromYaml) .config }}
-{{- if $config.service.pipelines.metrics }}
-{{- $_ := set $config.service.pipelines.metrics "receivers" (append $config.service.pipelines.metrics.receivers "k8s_cluster" | uniq)  }}
+{{- if index $config.service.pipelines "metrics/generic" }}
```

Comment:
Can you rename the "metrics/generic" to "metrics/internal" to be more explicit?

Reply:
done